app/views/shared/_request_status_information.html.erb

```diff
@@ -0,0 +1,37 @@
+<dl class='dl-horizontal dl-invert'>
+  <% if request.holdings.many? %>
+    <dt><%= t('forms.labels.holdings') %></dt>
+    <% request.holdings.each do |holding| %>
+      <dd><%= holding.callnumber %></dd>
+    <% end  %>
+  <% end %>
+
+  <% if request.destination.present? %>
+    <dt>
+      <%= label_for_pickup_libraries_dropdown(request.library_location.pickup_libraries) %>
+    </dt>
+    <dd>
+      <%= LibraryLocation.library_name_by_code(request.destination) %>
+    </dd>
+  <% end %>
+
+  <% if request.commentable? && request.data['comments'].present? %>
+    <dt><%= t("forms.labels.comments") %></dt>
+    <dd><%= request.data['comments'] %></dd>
+  <% end %>
+
+  <% if request.data.present? %>
```

Comment:
Ah, okay.  Since there's a few things flying around wrt the kinds of data we're storing and what we call it (see #121) maybe we can wait until the dust settles on that and then add all the appropriate accessors and tests.  Would that be reasonable?